tools/perf/benchmark_torchvision.py

```diff
@@ -11,14 +12,30 @@
 parser.add_argument('-b', '--batch-size', type=int, default=64)
 parser.add_argument('--fp16', action='store_true', default=False)
 parser.add_argument('-i', '--iter', type=int, default=100)
+parser.add_argument('-f', '--nhwc', type=bool, default=False)
```

Comment:
```suggestion
parser.add_argument('--nhwc', action='store_true', default=False)
```
So you just have to add `--nhwc` to the command rather than `--nhwc=True`